alpine/Dockerfile

```diff
@@ -16,10 +16,22 @@ RUN adduser -Su 1337 kong \
 	&& rm -rf /tmp/usr \
 	&& cp -R /tmp/etc / \
 	&& rm -rf /tmp/etc \
-	&& apk del .build-deps
+	&& apk del .build-deps \
+	&& chown -R kong /usr/local/kong \
+	&& chgrp -R 0 /usr/local/kong \
+	&& chmod -R g=u /usr/local/kong
+
+# for transparent proxy usage enable setcup for kong
+RUN chown kong /usr/local/openresty/nginx/sbin/ \
+	&& cp /usr/local/openresty/nginx/sbin/nginx /usr/local/openresty/nginx/sbin/nginx-transparent \
+	&& cp /usr/local/openresty/nginx/sbin/nginx /usr/local/openresty/nginx/sbin/nginx-non-transparent \
+	&& rm /usr/local/openresty/nginx/sbin/nginx \
+    && setcap cap_net_raw=+ep /usr/local/openresty/nginx/sbin/nginx-transparent
```

Comment:
This was the problem a few PRs ago: docker don't allow official images to rely on setcap.
That thread of conversation starts here: https://github.com/docker-library/official-images/pull/5180#issuecomment-447150620

Reply:
@james-callahan is correct. 

To shorten some github threads official images *will* allow setcap inside the docker entrypoint script and inside a conditional check.

Prior art here: [consul](https://github.com/hashicorp/docker-consul/blob/663f2cb7edbd7c8e7629b520cfc1578cd43f1a8a/0.X/docker-entrypoint.sh#L91-L93) and [vault](https://github.com/hashicorp/docker-vault/blob/6ca0fa239f6a7150dfab827a0852a0031a02e028/0.X/docker-entrypoint.sh#L83-L92)

Reply:
Also see https://github.com/docker-library/official-images/pull/5209#issuecomment-448827688 and https://github.com/docker-library/official-images/pull/5209#issuecomment-450449348

Reply:
Wasn't that problem already solved with https://github.com/docker-library/official-images/pull/5209?

Reply:
@subnetmarco it was; but that apparently broke openshift. Hence this PR.